pgxn/neon/libpagestore.c

```diff
@@ -623,7 +623,7 @@ pageserver_connect(shardno_t shard_no, int elevel)
 		shard->delay_us = MIN_RECONNECT_INTERVAL_USEC;
 
 		neon_shard_log(shard_no, DEBUG5, "Connection state: Connected");
-		neon_shard_log(shard_no, LOG, "libpagestore: connected to '%s' with protocol version %d", connstr, neon_protocol_version);
+		neon_shard_log(shard_no, DEBUG1, "libpagestore: connected to '%s' with protocol version %d", connstr, neon_protocol_version);
```

Comment:
It makes sense to me to not log this on every connection, but I'd still like to know from the logs which pageservers we're connecting to (especially if we later get an error that tells us a shard number but not the address).  This is really useful in a log, because the mapping of shard to pageserver changes through time.

If we would log the shard->pageserver mapping on startup and each time we're reconfigured, that would probably do the trick?  (perhaps we already do)

Reply:
+1